jax/experimental/jax2tf/jax2tf.py

```diff
@@ -1167,6 +1169,16 @@ def _sort(*operand: TfVal, dimension: int, is_stable: bool, num_keys: int) -> Tu
 
 tf_impl[lax.sort_p] = _sort
 
+def _fft(x, fft_type, fft_lengths):
+  tf_funcs = {FftType.FFT: [tf.signal.fft, tf.signal.fft2d, tf.signal.fft3d],
+              FftType.IFFT: [tf.signal.ifft, tf.signal.ifft2d, tf.signal.ifft3d],
+              FftType.RFFT: [tf.signal.rfft, tf.signal.rfft2d, tf.signal.rfft3d],
+              FftType.IRFFT: [tf.signal.irfft, tf.signal.irfft2d, tf.signal.irfft3d]}
+
+  return tf_funcs[fft_type][len(fft_lengths) - 1](x)
```

Comment:
Let's discuss this offline: the only thing we need from `fft_lengths` is its length?

Reply:
We had discussed online that we would add an assertion that `fft_lengths` has the expected values.